tests/BasicSearchTests.FunctionalTests.Core/BaseFunctionalTests.cs

```diff
@@ -21,6 +21,7 @@ public BaseFunctionalTests()
         public BaseFunctionalTests(string baseUrl)
         {
             // Arrange
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
This is now necessary as otherwise the Azure Search API will close the connection.

Reply:
Please do `|=`

Reply:
Oh whoops. Nice catch